pokemongo_bot/cell_workers/spin_fort.py

```diff
@@ -100,8 +105,9 @@ def work(self):
                 result = c.fetchone()
                 c.execute("SELECT DISTINCT COUNT(pokestop) FROM pokestop_log WHERE dated >= datetime('now','-1 day')")
                 if c.fetchone()[0] >= self.config.get('daily_spin_limit', 2000):
-                    self.emit_event('spin_limit', formatted='WARNING! You have reached your daily spin limit')
-                    sys.exit(2)
+                   if datetime.now() >= self.next_update:
+                       self.emit_event('spin_limit', formatted='WARNING! You have reached your daily spin limit')
+                       self._compute_next_update()
```

Comment:
Should stop running work here. Maybe add return WorkerResult.ERROR (or SUCCESS depending on your interpretation of this result)